test/integration/smoke/test_kubernetes_supported_versions.py

```diff
@@ -45,7 +45,7 @@ def setUpClass(cls):
         cls.services = cls.testClient.getParsedTestDataConfig()
         cls.zone = get_zone(cls.apiclient, cls.testClient.getZoneForTests())
         cls.mgtSvrDetails = cls.config.__dict__["mgtSvr"][0].__dict__
-        cls.kubernetes_version_iso_url = 'http://download.cloudstack.org/cks/setup-1.24.0.iso'
+        cls.kubernetes_version_iso_url = 'http://download.cloudstack.org/cks/setup-1.26.0.iso'
```

Comment:
should we get this from test_data as well?

Reply:
this should be read from test_data.  I will change it

Reply:
done